platform/features/layout/bundle.js

```diff
@@ -305,6 +283,27 @@ define([
                     "implementation": LayoutCompositionPolicy
                 }
             ],
+            "toolbars": [
+                {
+                    name: "Display Layout Toolbar",
+                    key: "layout",
+                    description: "A toolbar for objects inside a display layout.",
+                    forSelection: function (selection) {
+                        // Apply the layout toolbar if the selected object is inside a layout.
+                        return (selection && selection[1] && selection[1].context.item.type === 'layout');
+                    },
+                    toolbar: function (selection) {
+                        return [
```

Comment:
This is still very much tied to the old API, which we will need to revisit during refactoring. 

I wonder if perhaps we should take an approach similar to the [IndicatorsAPI]() here where either a simple control with some customizable options can be provided (based on however our custom form controls end up being defined), or an arbitrary DOM element can be provided.

Reply:
This is a legacy patch that allows us to refactor toolbars without having to convert bundles to plugins. The “public” side of this is exposed separate from the legacy api. 

The toolbar api is still draft after this merges, and I expect some additions to support custom controls, but that should be tackled during our forms overhaul.